### PR TITLE
Fixed a few grammatical errors

### PR DESCRIPTION
**Pull request by ForgottenProgramme**
Fixed a few grammatical errors on the introduction page of the documentation.

### CODE REVIEW

**Review by beckermr — CHANGES_REQUESTED** [review]
You need the edit the docs in the src dir. The docs dir is what is generated by the build.

**Comment by ForgottenProgramme** [comment]
@beckermr , 
oh, okay.
Thanks Mathew.

**Comment by beckermr** [comment]
Thank you for the PR!

**Comment by ForgottenProgramme** [comment]
Created a new PR with suggested changes.

**Comment by ocefpaf** [comment]
> You need the edit the docs in the src dir. The docs dir is what is generated by the build.

Let me rephrase that. We need docs on how to edit the docs ;-p

We all made that mistake at some point.

**Comment by ForgottenProgramme** [comment]
@ocefpaf 
:D
>We need docs on how to edit the docs 

Noted!

**Comment by beckermr** [comment]
I updated the pull request template to make this clearer. It was already in there but now it is in a check list.
